modules/adagioBidAdapter.js

```diff
@@ -1,58 +1,44 @@
-import {find} from '../src/polyfill.js';
+import { BANNER, NATIVE, VIDEO } from '../src/mediaTypes.js';
 import {
   canAccessWindowTop,
   cleanObj,
   deepAccess,
   deepClone,
   generateUUID,
   getDNT,
-  getUniqueIdentifierStr,
   getWindowSelf,
   getWindowTop,
   isArray,
   isArrayOfNums,
   isFn,
-  inIframe,
   isInteger,
   isNumber,
   isSafeFrameWindow,
   isStr,
   logError,
   logInfo,
   logWarn,
-  mergeDeep,
 } from '../src/utils.js';
-import {config} from '../src/config.js';
-import {registerBidder} from '../src/adapters/bidderFactory.js';
-import {loadExternalScript} from '../src/adloader.js';
```

Comment:
please remove your bid module from adloader.js whitelist

Reply:
I didn't remove it because the Rtd Provider use the same name `"adagio"`

https://github.com/prebid/Prebid.js/blob/master/modules/adagioRtdProvider.js#L33

Reply:
Btw, I should move it from "Bid Modules" to "RTD Modules" section.